package.json

```diff
@@ -4,18 +4,23 @@
   "license": "MIT",
   "description": "generates a cryptographically strong random uid",
   "repository": "vercel/uid-promise",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.js",
+  "types": "lib/cjs/index.d.ts",
   "files": [
     "lib"
   ],
   "scripts": {
-    "build": "tsc",
-    "test": "tsc && ava"
+    "build": "yarn build:cjs && yarn build:esm",
+    "build:cjs": "tsc",
+    "build:esm": "tsc --module esnext --outDir ./lib/esm",
+    "test": "yarn test:cjs && yarn test:esm",
+    "test:cjs": "yarn build:cjs && ava",
+    "test:esm": "yarn build:esm && ava --node-arguments=\"--experimental-global-webcrypto\""
```

Comment:
Nit pick: this isn't testing esm, its testing cjs with the global crypto

Reply:
:hmmyes: